contrib/test_jenkins.sh

```diff
@@ -959,8 +959,10 @@ run_ucp_client_server() {
 }
 
 run_io_demo() {
-	server_ip=$(get_rdma_device_ip_addr)
-	if [ "$server_ip" == "" ]
+	server_rdma_addr=$(get_rdma_device_ip_addr)
+	server_loopback_addr="127.0.0.1"
+
+	if [ "$server_rdma_addr" == "" ]
```

Comment:
if no rdma, should we test just on loopback? or some other TCP addr?

Reply:
I think we should definitely test rdma, it's the "most important" one..
the jenkins file doesn't have TCP addr testing. should i port the CM fallback testing to v1.10?

Reply:
for v1.10 it would be enough to test rdma+127.0.0.1
for master - need to test rdma,tcp,127.0.0.1. is this the situation today?

Reply:
yes: master - https://github.com/openucx/ucx/blob/master/contrib/test_jenkins.sh#L1003